content/blog/2018/01/2018-01-03-fosdem-2018.adoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "FOSDEM 2018!"
+tags:
+- community
+author: markewaite
+---
+
+link:https://fosdem.org/2018/[FOSDEM 2018] is a free event for software developers to meet, share ideas and collaborate.
+It is an annual event that brings open source contributors from around the world for two days of presentations, discussions, and learning.
+
+Jenkins will be well-represented at FOSDEM 2018.
+
+image:/images/post-images/2018-01-03-fosdem/fosdem-2018.png[FOSDEM 2018, role=center]
+
+## Happy Hour before FOSDEM
+
+We'll have a link:https://www.meetup.com/jenkinsmeetup/events/245685813/[happy hour] Friday evening before FOSDEM at Cafe Le Roy d'Espagne.
+See the link:https://www.meetup.com/jenkinsmeetup/events/245685813/[meetup page] for details.
+
+## Jenkins table at FOSDEM
+
+A Jenkins table will be staffed by volunteers at FOSDEM to answer questions, discuss topics, and help users.
+See the link:https://www.meetup.com/jenkinsmeetup/events/245688007/[meetup page] for details.
+
+## Presentations at FOSDEM
+
+* link:https://fosdem.org/2018/schedule/event/containers_automated_deployments/[Automated Linux Containers deployment for fun and profit] by link:https://fosdem.org/2018/schedule/speaker/david_negreira/[David Negreira]
+* link:https://fosdem.org/2018/schedule/event/advanced_testing_java/[Advanced testing in action on a Java project] by link:https://fosdem.org/2018/schedule/speaker/vincent_massol/[Vincent Massol]
+* link:https://fosdem.org/2018/schedule/event/deployment_provisioning_orchestration/[Deployment vs Provisioning vs Orchestration vs Configuration Management] by link:https://fosdem.org/2018/schedule/speaker/peter_souter/[Peter Souter]
+
+## Jenkins Hackfest after FOSDEM
```

Comment:
After the @alyssat's response in the ML I am still not sure whether there will be a "contributor summit" part. LGTM as is though

Reply:
I intentionally left the contributor summit off the list as a result of a conversation with @alyssat .  It can be added if needed.